proposals/incubation/credebl.md

```diff
@@ -0,0 +1,137 @@
+
+Once these final steps are completed, we believe the project will be ready for "Graduated" status and will apply for
+this designation immediately.
+
+We anticipate that this designation process will proceed swiftly if the TAC concurs that all criteria have been met.
+
+
+# Solution
+The CREDEBL Codebase is currently hosted as a separate Github Org at https://github.com/credebl
+
+### CREDEBL PLATFORM
+Platform provides a set of scalable services to manage decentralized identity and verifiable credentials (VC) system. It enables developers to build Self-Sovereign Identity (SSI) solutions that provide enhanced privacy and security. It is designed for scalability to support population-scale SSI implementations. 
+
+It provides tools for digital credential management and verification, allowing organizations to easily issue and manage credentials with ease. It is built on a micro-services architecture to support large-scale enterprise applications.
+
+Platform supports multiple ledger options (like Indy and Polygon) and also ledger-less credential issuance using methods like did:web, did:key and did:peer.
+
+### CREDEBL STUDIO
+Studio is web front-end for the entire Platform, built using Astro, React, Flowbite, and Tailwind CSS, allowing administrators & users to easily access & manage all the Platform features using a simple web interface.
+
+### ARIES AGENT
+In the context of decentralized identity systems, an agent generally refers to a software component or service that facilitates peer-to-peer interactions and data exchange operations between various entities in the system.
+
+Agents leverage the Aries protocol implementations built into Credo & Aca-Py.
+
+There are two types of agents that organizations can use, each serving different needs based on the deployment model and use case: Shared Agents and Dedicated Agents.
+
+A shared agent refers to an agent that is used by multiple entities or tenants, rather than being dedicated to a single entity. If the tenancy is set to true, it typically means that the system or agent supports multi-tenancy, or if the tenancy is set to false, it means that the agent does not support the multi-tenancy.
+
+### Mobile SDK
+Built on Credo, the Mobile Wallet SDK is a React-Native SDK wrapper that helps mobile developers to easily build SSI features in the existing mobile apps or a new one. It enables interoperability and seamless connectivity for secure data exchange.
+
+# Architecture
+Following diagram depicts the overall system architecture of CREDEBL.
+
+All the listed components in this diagram are proposed to be in the scope of contribution under LF Decentralized Trust.
+
+![CREDEBL Architecture](../images/credebl/CREDEBL_OSS_Architecture.png "CREDEBL Architecture")
+
+# Relation with existing identity projects
+CREDEBL closely relates to the existing Hyperledger Identus project, and shares many concepts such as use of Aries RFCs, DIF & TrustOverIP specifications.
+
```

Comment:
How does it relate to Janssen Project?

Reply:
There was a question - whether a Linux Foundation project can also be a recognised DPG (Digital Public Good) by DPG Alliance? Janssen Project is hence mentioned as a reference [being a DPG](https://gluu.org/janssen-project-is-a-digital-public-good/) as well as a Linux Foundation project.